modules/programs/rofi.nix

```diff
@@ -240,6 +247,7 @@ in
   };
 
   config = mkIf cfg.enable {
+    warnings = optional (cfg.theme != null && cfg.colors != null) "rofi: colors shouldn't be set when using themes";
```

Comment:
If setting colors directly will be removed from rofi in the future, perhaps it is good to deprecate the option completely?